src/SAWScript/Interpreter.hs

```diff
@@ -2392,6 +2393,13 @@ primitives = Map.fromList
     (pureVal disable_crucible_profiling)
     Current
     ["Stop recording profiling information."]
+
+  , prim "summarize_verification" "TopLevel ()"
+    (pureVal summarize_verification)
+    Experimental
+    [ "Provide a human-readable summary of all verifications performed"
```

Comment:
nit: "Print a human-readable summary of all verification tasks performed."

Reply:
Fixed.